test/resources/auth.js.tmp

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const testingHeaders = {
+  'X-Watson-Learning-Opt-Out': 1,
+  'X-Watson-Test': 1,
+};
+
+module.exports = {
+  assistant: {
+    url: 'http://localhost:3000/assistant/api',
+    iam_apikey: '_CVCfS14pvk6yni9qDK7UKZtgjZojLDIDObGBmENRWAg',
+    workspace_id: 'dc762f48-e10c-4418-92c1-c7be6feadbc0',
+    assistant_id: '3920829d-69e9-4384-aa23-40b94d7218f3',
+    headers: Object.assign(testingHeaders, { 'x-watson-origin': 'assistant-test' }),
+  },
+  compare_comply: {
+    url: 'http://localhost:3000/compare_comply/api',
+    iam_apikey: 'XjM8fK9vLj9f9gQPJsqNrYzfPnTiHW8fYQUG2-Rr8wLN',
+    headers: testingHeaders,
+    version: '2018-12-06',
+  },
+  discovery: {
+    iam_apikey: 'yLxh4uy-gzm7AkAplwXCwYPOhLwnq5VRThk8jM5K-8zz',
+    environment_id: '62b0dd87-eefa-40bf-81d6-cf9bc82692ab',
+    configuration_id: 'a23e6df9-5acd-4cc5-be71-a24f5faf9b9a',
+    collection_id: 'c44a8fd0-5fe1-481f-905c-a9d5726ab45e',
+    collection_id2: '083ccf4e-b09a-474e-83c8-fe459f061daa',
+    japanese_collection_id: 'bea2e04b-3b93-4422-95ae-aafe6d6b2244',
+    version_date: '2018-10-24',
+    headers: testingHeaders,
+  },
+  language_translator: {
+    url: 'http://localhost:3000/lang_trans/api',
+    iam_apikey: 'y-vjnhjGNuh64I4RqLCHjgVutVMxo4ZVivTt7MoLLPFn',
+    headers: testingHeaders,
+    version: 'v3',
+  },
+  natural_language_classifier: {
+    url: 'http://localhost:3000/nat_lang_classify/api',
+    iam_apikey: 'UuhYsd-ACLwOKnwtDYPF1gFvYeFSAC0FZ5tpK3DIMOX6',
+    classifier_id: '084b83x538-nlc-558',
+    headers: testingHeaders,
+    version: 'v1',
+  },
+  natural_language_understanding: {
+    iam_apikey: 'i9H3JaifgrMeVZbCad9WSZOI_WDcrHbd5ynSAtOpd_p4',
+    url: 'http://localhost:3000/nat_lang_understand/api',
+    headers: testingHeaders,
+  },
+  personality_insights: {
+    url: 'http://localhost:3000/person_ins/api',
+    iam_apikey: 'sPTvflqVlpasoQN-TD11ukn578aC0aIa2bmhwVKYP3hM',
+    headers: testingHeaders,
+    version: 'v3',
+    version_date: '2016-10-19',
+  },
+  speech_to_text: {
+    url: 'http://localhost:3000/s2t/api',
+    iam_apikey: 'JSsIxpr6aHwHhTicbeFlA9X_XdBD2rd5BM6WZD2YDnjN',
+    headers: testingHeaders,
+    version: 'v1',
+  },
+  text_to_speech: {
+    url: 'http://localhost:3000/t2s/api',
+    iam_apikey: 'OldPvSWgwIMXs_FcCkDc-MqX3cUiZgC1gop0yYTPipnk',
```

Comment:
hopefully these are not real credentials? how did you push this without getting an error from the detect secret package?